opm/simulators/wells/WellInterface_impl.hpp

```diff
@@ -1201,13 +1201,15 @@ namespace Opm
                               WellState& well_state,
                               Opm::DeferredLogger& deferred_logger)
     {
-        const int max_iter = param_.max_welleq_iter_;
+        const int max_iter = param_.max_inner_iter_wells_;
         int it = 0;
-        const double dt = 1.0; //not used for the well tests
+        const double dt = ebosSimulator.timeStepSize();
```

Comment:
not sure what is the best `dt` for well testing. And I am not against the current solution. For most of cases, it should not make big difference.  The change within this function makes us have to test this PR with prediction cases.